custom_components/ramses_cc/broker.py

```diff
@@ -186,7 +186,12 @@ def new_entities(
         new_zones = new_entities(
             self._zones,
             set().union(
-                *[s.zones for s in self.client.systems if isinstance(s, MultiZone)]
+                *[s.zones for s in self.client.systems if isinstance(s, MultiZone)],
+                [
```

Comment:
There should be an `s.child` - but it wouldn't be too reliable - so let's stick with this - thanks!